src/app/services/console-service.ts

```diff
@@ -0,0 +1,11 @@
+import { Injectable } from '@angular/core';
+
+@Injectable()
+export class ConsoleService {
+
+    public consoleArray: Array<string> = new Array;
```

Comment:
Mimo że zapis `new Array` działa, częściej stosuje się jednak tworzenie obiektów z nawiasami: `new Array()`.
https://stackoverflow.com/questions/3034941/can-we-omit-parentheses-when-creating-an-object-using-the-new-operator